theia-plugin/lib/theia-plugin-backend.js.map

```diff
@@ -0,0 +1 @@
+{"version":3,"file":"theia-plugin-backend.js","sourceRoot":"","sources":["../src/theia-plugin-backend.ts"],"names":[],"mappings":";AACA;;GAEG;;AAEH,qCAAuC;AAEvC,eAAsB,OAA4B;IAC9C,IAAM,6BAA6B,GAAG;QAClC,EAAE,EAAE,+BAA+B;QACnC,KAAK,EAAE,aAAa;KACvB,CAAC;IACF,OAAO,CAAC,aAAa,CAAC,IAAI,CAAC,KAAK,CAAC,QAAQ,CAAC,eAAe,CAAC,6BAA6B,EAAE;QAAC,cAAc;aAAd,UAAc,EAAd,qBAAc,EAAd,IAAc;YAAd,yBAAc;;QACpG,KAAK,CAAC,MAAM,CAAC,sBAAsB,CAAC,cAAc,CAAC,CAAC;IACxD,CAAC,CAAC,CAAC,CAAC;AAER,CAAC;AATD,sBASC;AAED;AAEA,CAAC;AAFD,oBAEC"}
```

Comment:
This file does not belong it the repo, sourcemaps should be ignored.

Reply:
The directory will be changed to `out` in #3:
https://github.com/eclipsesource/jsonforms-tooling/pull/3/files#diff-642a7e14f16e7c3b7018ed2564acc044R10

And all `out` folders will be ignored, so I think ignoring the `lib` folder is not necessary.  
But I'll delete the `lib` folder